README.md

```diff
@@ -136,23 +148,26 @@ Credits
 =======
 
 Template-Coq was originally developed by
-[Gregory Malecha](https://github.com/gmalecha), and is now developed by
-[Abhishek Anand](https://github.com/aa755), [Simon
-Boulier](https://github.com/simonboulier) and [Matthieu
-Sozeau](https://github.com/mattam82).
-
-Contributors include [Yannick Forster](https://github.com/yforster),
-[Cyril Cohen](https://github.com/CohenCyril) and [Nicolas
-Tabareau](https://github.com/Tabareau).
-
-Copyright (c) 2014-2018 Gregory Malecha\
-Copyright (c) 2015-2018 Abhishek Anand, Matthieu Sozeau\
-Copyright (c) 2017-2018 Simon Boulier, Nicolas Tabareau, Cyril Cohen
+[Gregory Malecha](https://github.com/gmalecha).
+
+MetqCoq and is now developed by [Abhishek
```

Comment:
Typo.

Reply:
Sorry, I had not started this branch on the right commit, I modified PR #156 which had the `README.md` changes, now removed from this one.